libcontainer/configs/validate/intelrdt_test.go

```diff
@@ -0,0 +1,122 @@
+package validate
+
+import (
+	"testing"
+
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func TestValidateIntelRdt(t *testing.T) {
+	// Call init to trigger the sync.Once and enable overriding the rdt status
+	intelRdt.init()
+
+	testCases := []struct {
+		name       string
+		rdtEnabled bool
+		catEnabled bool
+		mbaEnabled bool
+		config     *configs.IntelRdt
+		isErr      bool
+	}{
+		{
+			name:       "rdt not supported, no config",
+			rdtEnabled: false,
+			config:     nil,
+			isErr:      false,
```

Comment:
Let's skip the zero value, unless you want to highlight that in some case. I think here we can just skip these variables.

Let's err on the side of skipping setting variables to their empty value rather than being extra verbose.

Reply:
Done